kafka-ui-react-app/src/components/Topics/shared/Form/TopicForm.tsx

```diff
@@ -34,18 +36,17 @@ const TopicForm: React.FC<Props> = ({
           <div className="columns">
             <div className={`column ${isEditing ? '' : 'is-three-quarters'}`}>
               <label className="label">Topic Name *</label>
-              <input
-                className="input"
+              <Input
+                name="name"
                 placeholder="Topic Name"
                 defaultValue={topicName}
-                {...register('name', {
+                hookFormOptions={{
                   required: 'Topic Name is required.',
```

Comment:
Let's try to move validation to specific resolver, which handles validation of all fields
The `yup` validator can validate by multiple rules with different messages and can be defined outside the render function